server/src/controllers/Auth/resolver.ts

```diff
@@ -39,7 +39,18 @@ export class AuthResolver {
       throw new Error('EMAIL_IN_USE');
     }
 
-    const userData: Prisma.usersCreateInput = data;
+    const instanceMember = await prisma.instance_roles.findFirst({
+      where: { name: 'member' },
+    });
```

Comment:
```suggestion
    const instanceMember = await prisma.instance_roles.findUnique({
      where: { name: 'member' },
    });
```
I think we can tighten this, since the schema declares these as unique.

Reply:
Nice catch, this was before I added (missing) unique constraint on the `instance_roles` name.